manifests/addons/gen.sh

```diff
@@ -46,24 +46,33 @@ helm3 template prometheus stable/prometheus \
   -f "${WD}/values-prometheus.yaml" \
   > "${ADDONS}/prometheus.yaml"
 
+function compressDashboard() {
+  < "${DASHBOARDS}/$1" jq -c  > "${TMP}/$1"
+}
+
 # Set up grafana
 {
   helm3 template grafana stable/grafana \
     --namespace istio-system \
     --version 5.3.5 \
     -f "${WD}/values-grafana.yaml"
 
-  # Set up grafana dashboards. Split into 2 to avoid Kubernetes size limits
+  # Set up grafana dashboards. Split into 2 and compress to single line json to avoid Kubernetes size limits
+  compressDashboard "pilot-dashboard.json"
```

Comment:
Added compression to fit in k8s. This cuts the size by about 40% giving us some breathing room